src/website/views/custom-elements/popover.pug

```diff
@@ -168,178 +154,118 @@ ul#a-tabsb--positioning.a-tabs
   .example.-mb--3
     .-p--3
       button#example-4-t-button.a-btn.-mr--2.-mb--2.-mb-md--0 Top popover
-      chi-popover#example-4-t-popover(position="top", arrow, reference="#example-4-t-button")
-        .-px--2.-py--1
-          h5 Popover!
-          p This is popover is top positioned.
+      chi-popover#example-4-t-popover(position="top", title="Popover!", variant="text", arrow, reference="#example-4-t-button")
+        | This is popover is top positioned.
 
       button#example-4-ts-button.a-btn.-mr--2.-mb--2.-mb-md--0 Top start popover
-      chi-popover#example-4-ts-popover(position="top-start", arrow, reference="#example-4-ts-button")
-        .-px--2.-py--1
-          h5 Popover!
-          p This is popover is top-start positioned.
+      chi-popover#example-4-ts-popover(position="top-start", title="Popover!", variant="text", arrow, reference="#example-4-ts-button")
+        | This is popover is top-start positioned.
 
       button#example-4-te-button.a-btn.-mr--2.-mb--2.-mb-md--0 Top end popover
-      chi-popover#example-4-te-popover(position="top-end", arrow, reference="#example-4-te-button")
-        .-px--2.-py--1
-          h5 Popover!
-          p This is popover is top-end positioned.
+      chi-popover#example-4-te-popover(position="top-end", title="Popover!", variant="text", arrow, reference="#example-4-te-button")
+        | This is popover is top-end positioned.
 
     :code(lang="html")
       <button class="a-btn -mr--2 -mb--2 -mb-md--0" id="example-4-t-button">Top popover</button>
```

Comment:
Can we remove all these spacing utility classes from the Positioning and Interaction `code` examples? We're using them to provide sufficient padding in the documentation example but they're not relevant to the component.

Reply:
Makes sense, done!

Reply:
I also changed the way CE methods are displayed for all the custom elements (Now they are shown using a table, similar to Properties and Events).